### ISSUE TITLE
[Accessibility]: Command palette icon is not focusable

### ISSUE THREAD

**Issue by tjkandala** [sourcegraph/sourcegraph#36859]
### Audit type

Keyboard navigation

### User journey audit issue

#34061

### Problem description

When tabbing through the global navbar, the command palette icon is not focusable. The only ways to open the command palette are through mouse click and the `cmd` + `p` shortcut.

### Expected behavior

The command palette icon should be focusable.

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by sourcegraph-bot-2** [comment]
Heads up @muratsu @jjinnii @ryankscott - the "team/integrations" label was applied to this issue.

**Comment by umpox** [comment]
Closed by https://github.com/sourcegraph/sourcegraph/pull/37523
